pkg/binlog/position.go

```diff
@@ -172,6 +177,21 @@ type Location struct {
 	GTIDSet gtid.Set
 }
 
+// NewLocation returns a new Location
+func NewLocation(flavor string) Location {
+	if flavor == gmysql.MariaDBFlavor {
+		return Location{
+			Position: MinPosition,
+			GTIDSet:  &gtid.MariadbGTIDSet{},
```

Comment:
How about giving `GTIDSet.set` an empty value rather than `nil`? (`gmysql.ParseMariadbGTIDSet("")`)? ditto for L191 and `gtid. MinGTIDSet`.

Reply:
update in 8132f8f